test/image-blueprints/layer2-presubmit/group1/rhel94-source-base.toml

```diff
@@ -0,0 +1,39 @@
+name = "rhel-9.2-microshift-source-base"
```

Comment:
Like in my previous comment, naming convention is a bit confusing.
Should this be `rhel-9.4`?

Reply:
In general, file name should correspond to what's built inside.